searx/plugins/tracker_url_remover.py

```diff
@@ -0,0 +1,40 @@
+'''
+searx is free software: you can redistribute it and/or modify
+it under the terms of the GNU Affero General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+searx is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU Affero General Public License for more details.
+
+You should have received a copy of the GNU Affero General Public License
+along with searx. If not, see < http://www.gnu.org/licenses/ >.
+
+(C) 2015 by Adam Tauber, <asciimoo@gmail.com>
+'''
+
+from flask.ext.babel import gettext
+import re
+
+re1 = re.compile(r'utm_[^&]+&?')
+re2 = re.compile(r'(wkey|wemail)[^&]+&?')
```

Comment:
^^^ replaces e.g. `http://wkey.com/` to `http://'
maybe checking the query params from `result['parsed_url']` would be a safer solution